templates/appliance/plex.html

```diff
@@ -0,0 +1,171 @@
+      </ul>
+    </div>
+    <div class="col-4">
+      <h2 class="p-heading--three">Snaps in the image</h2>
+      <div class="p-media-object">
+        <img src="https://assets.ubuntu.com/v1/402069bd-plex-media-server.png?w=75" class="p-media-object__image is-round" height="75" width="75">
+        <div class="p-media-object__details">
+          <h3 class="p-media-object__title">
+            <a href="https://snapcraft.io/plexmediaserver" class="p-link--external">Plex Media Server</a>
+          </h3>
+          <p class="p-media-object__content">By Plex, Inc. (plexinc)</p>
+          <ul class="p-media-object__meta-list">
+            <li>
+              <h4 class="p-muted-heading">Channel</h4>
+              <p>latest/stable</p>
+            </li>
+            <li>
+              <h4 class="p-muted-heading">Version</h4>
+              <p>1.19.3.2764-ef515a800</p>
+            </li>
+            <li>
+              <h4 class="p-muted-heading">Published</h4>
+              <p>29 April 2020</p>
+            </li>
+          </ul>
+        </div>
+      </div>
+      <hr>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip--light">
+  <div class="row">
+    <div class="col-8">
+      <h2>Appliance policies</h2>
+      <p>Every Ubuntu appliance publisher is responsible for providing privacy policies and rights to operate with their brand. More information can be found on the governance page.</p>
+      <a href="https://www.plex.tv/en-gb/about/privacy-legal/" class="p-button--neutral p-link--external">Plex Media Server Policies</a>
+    </div>
+    <div class="col-4 u-align--center">
+      <img src="https://assets.ubuntu.com/v1/cb5ac5c9-privacy-policy.svg" alt="" height="200" width="200">
+    </div>
+  </div>
+</section>
+
+<section class="p-strip">
+  <div class="row">
+    <div class="col-8">
+      <h2>The developers</h2>
+      <p>The Plex media Ubuntu appliance is a commercial appliance and the software is produced by developers at Plexinc.</p>
+      <a href="https://plex.tv" class="p-button--neutral p-link--external">Plex.tv</a>
+    </div>
+    <div class="col-4 u-align--center u-vertically-center">
+      <img src="https://assets.ubuntu.com/v1/3cd85693-Plex.png" alt="" height="100" width="250">
+    </div>
+  </div>
+</section>
+
+<section class="p-strip--light">
+  <div class="row">
+    <div class="col-8">
+      <h2>Get started</h2>
+      <p>Once you have installed and set up your Plex Media Server Ubuntu appliance you will be taken through set up in your browser. If you have any more questions or want to go further, the developers of Plex Media Server have lots of documentation to help you get started.</p>
+      <a href="https://snapcraft.io/plexmediaserver" class="p-button--neutral p-link--external">Get Started</a>
+    </div>
+    <div class="col-4 u-align--center u-vertically-center">
+      <img src="https://assets.ubuntu.com/v1/402069bd-plex-media-server.png?w=150" height="150" width="150">
```

Comment:
Should this one be w=200 like the others above?

Reply:
Being round, it looked a bit huge at 200. I'll see what design think